test-resources-r2dbc/test-resources-r2dbc-oracle-xe/src/test/groovy/io/micronaut/testresources/r2dbc/oracle/WithJdbcStartOracleTest.groovy

```diff
@@ -0,0 +1,31 @@
+package io.micronaut.testresources.r2dbc.oracle
+
+import io.micronaut.test.extensions.spock.annotation.MicronautTest
+import io.micronaut.testresources.jdbc.AbstractJDBCSpec
+import io.micronaut.testresources.jdbc.Book
+import jakarta.inject.Inject
+import spock.lang.Ignore
+
+@MicronautTest(environments = ["jdbc"] )
+@Ignore("R2dbcSchemaGenerator doesn't support Oracle")
```

Comment:
I think we could. I got an error when running the test so didn't want to spend time debugging R2DBC Oracle support. Is there an easy way to provide a schema without having to rely on Flyway?

Reply:
Perhaps using Groovy SQL?

Reply:
Looks like I wouldn't even be able to get past it, since if I don't use schema creation, I get another error which seems to indicate that the driver isn't stable:

```
'java.util.concurrent.Flow$Publisher oracle.jdbc.OracleConnectionBuilder.buildConnectionPublisherOracle()'
java.lang.NoSuchMethodError: 'java.util.concurrent.Flow$Publisher oracle.jdbc.OracleConnectionBuilder.buildConnectionPublisherOracle()'
	at app//oracle.r2dbc.impl.OracleReactiveJdbcAdapter.lambda$publishConnection$10(OracleReactiveJdbcAdapter.java:687)
	at app//oracle.r2dbc.impl.OracleR2dbcExceptions.fromJdbc(OracleR2dbcExceptions.java:279)
	at app//oracle.r2dbc.impl.OracleReactiveJdbcAdapter.lambda$deferOnce$26(OracleReactiveJdbcAdapter.java:1102)
	at app//reactor.core.publisher.FluxSource.subscribe(FluxSource.java:67)
	at app//reactor.core.publisher.Flux.subscribe(Flux.java:8469)
	at app//oracle.r2dbc.impl.AsyncLock.lambda$lock$4(AsyncLock.java:203)
	at app//oracle.r2dbc.impl.AsyncLock.lock(AsyncLock.java:99)
	at app//oracle.r2dbc.impl.AsyncLock.lambda$lock$5(AsyncLock.java:202)
```

So let's keep as is for now.